contrib/CONTRIBUTING.md

```diff
@@ -17,11 +17,15 @@ The following is a set of guidelines for contributing to the Bittensor ecosystem
 
```

Comment:
since you are here - do you mind to move the CONTRIBUTING documentation either at the top-level of the repository? In this way it gets picked up properly by Github, from their docs:

> To help your project contributors do good work, you can add a file with contribution guidelines to your project repository's root, docs, or .github folder. 

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#about-contributing-guidelines

In this way it would be showed directly in https://github.com/masa-finance/masa-bittensor/contribute

Reply:
done myself from the GH UI